contracts/truefi2/SpotBaseRateOracle.sol

```diff
@@ -36,4 +44,10 @@ contract SpotBaseRateOracle {
         (, , , , uint128 currentVariableBorrowRate, , , , , , , ) = aaveLendingPool.getReserveData(asset);
         return uint256(currentVariableBorrowRate).div(1e23);
     }
+
+    function _getCompoundBorrowAPY(address asset) internal view returns (uint256 apy) {
+        uint256 borrowRatePerBlock = ICToken(asset).borrowRatePerBlock();
+        uint256 exponent = borrowRatePerBlock.mul(blocksPerDay).div(1e18).add(1);
```

Comment:
Because of the solidity math, this will always result in 1, which leads the following line to return always 0.